crates/ruff_linter/src/rules/pydocstyle/rules/sections.rs

```diff
@@ -78,9 +78,6 @@ use crate::rules::pydocstyle::settings::Convention;
 ///         raise FasterThanLightError from exc
 /// ```
 ///
-/// ## Options
-/// - `lint.pydocstyle.convention`
-///
```

Comment:
I think we should keep this reference because changing the convention might disable this rule (it's not enabled for PEP257)

https://github.com/astral-sh/ruff/blob/83b1c48a935f17869b9ee618faadc99213f83d95/crates/ruff_linter/src/rules/pydocstyle/settings.rs#L55-L73

Reply:
Apologies. I was basing the need for the link to the convention setting based on whether it was documented within the main rule explanation. I have now fixed that documentation as well, for consistency.